editor/inspector_dock.cpp

```diff
@@ -86,6 +87,15 @@ void InspectorDock::_menu_option_confirm(int p_option, bool p_confirmed) {
 		case RESOURCE_EDIT_CLIPBOARD: {
 			_paste_resource();
 		} break;
+		case RESOURCE_SHOW_IN_FILESYSTEM: {
+			ObjectID current_id = EditorNode::get_singleton()->get_editor_selection_history()->get_current();
+			Object *current_obj = current_id.is_valid() ? ObjectDB::get_instance(current_id) : nullptr;
+
+			ERR_FAIL_COND(!Object::cast_to<Resource>(current_obj));
+
+			Ref<Resource> current_res = Ref<Resource>(Object::cast_to<Resource>(current_obj));
```

Comment:
Is `id` check necessary? `get_instance` should work with null ID, so it's probably will work like this:
```suggestion
			Ref<Resource> current_res = Ref<Resource>(Object::cast_to<Resource>(ObjectDB::get_instance(current_id)));

			ERR_FAIL_COND(current_res.is_null());
```

Reply:
This code is used in a few places in this file, I just copied it.
See #70923